src/komponenter/kalkulator/beregningerForAGP2.ts

```diff
@@ -118,6 +119,32 @@ const getInformasjonOmAGP2HvisAGP2IkkeErNåddOgPermitteringIkkeErLøpende = (
     };
 };
 
+export const getInformasjonOmAGP2HvisAGP2NåsIFramtidenUtenLøpendePermittering = (
```

Comment:
Enig i denne implementasjonen. Den er faktisk basically helt lik `finnDatoAGP2LøpendePermittering` på branchen der jeg jobber med løpende intervaller! 🤯 
https://github.com/navikt/arbeidsgiver-permittering/pull/485/files#diff-002dd0b949826ff39f6df7661959f380cd92df720b7e0c0ae9ff783ed8b694e2R180
Jeg tror vi kan forenkle logikken i `finnInformasjonAGP2` en del når vi merger disse to casene.